tools.go

```diff
@@ -19,11 +19,8 @@ package jaeger
 
 // https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 import (
-	_ "honnef.co/go/tools/cmd/staticcheck"
-
+	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	_ "github.com/mjibson/esc"
-	_ "github.com/securego/gosec/cmd/gosec"
```

Comment:
indirect dependency of `golangci-lint` will be naturally included in `go.mod` as indirect dependency.